src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -183,6 +185,10 @@ public function denormalize($data, $class, $format = null, array $context = arra
             }
 
             if (($allowedAttributes !== false && !in_array($attribute, $allowedAttributes)) || !$this->isAllowedAttribute($class, $attribute, $format, $context)) {
+                if (isset($context['extra_attributes']) && !$context['extra_attributes']) {
```

Comment:
The form component has a similar option called `allow_extra_fields`. I propose to rename the key to `allow_extra_attributes` for consistency.